src/App.js

```diff
@@ -8,19 +8,32 @@ import Profile from './components/profile/Profile';
 import { Routes, Route, Navigate, Outlet } from 'react-router-dom';
 
 function App() {
-  const [profileData] = useState({
+
+  const [user, setUser] = useState({
     first_name: "Nathan",
     last_name: "King",
     biography: "Hello world",
     github_url: "https://github.com/vherus"
   })
 
+  const [token] = useState(`Bearer ${process.env.REACT_APP_USER_TOKEN}`)
+
+  useEffect(() => {
+    fetch(`${process.env.REACT_APP_API_URL}/user/4`, {
+      method: "GET",
+      headers: { Authorization: token }
```

Comment:
You should be getting the token from localStorage, the token is saved in there after the user logs in

`Authorization: "Bearer " + localStorage.getItem(process.env.REACT_APP_USER_TOKEN)`